### ISSUE TITLE
Three concerns about “Focus indicator size sufficient”

### ISSUE THREAD

**Issue by julierawe** [w3c/wcag3#809]
[Focus indicator size sufficient](https://w3c.github.io/wcag3/guidelines/#focus-indicator-size-sufficient) says “If a custom [focus indicator](https://w3c.github.io/wcag3/guidelines/#dfn-focus-indicator) is used, it has sufficient size and adjacency.” 
								
We have a few questions about this provision:

- **Requirement level:** Why is this supplemental? It should be core that people can see it. Maybe there should be a minimum at the core level and an enhanced version? 
- **Adjacency:** What is this provision saying about adjacency? In the current draft, neither the provision nor the tests say anything about adjacency. 
- **Definition:** Why doesn’t the [WCAG 3 definition of “focus indicator”](https://w3c.github.io/wcag3/guidelines/#dfn-focus-indicator) mention “perimeter” or “bounding box” or “ring” or “border” around the element that is receiving focus? 

    - We are concerned that the current definition only says “Pixels changed.” This wording leaves open the possibility that users may not know where the center of focus is.
    - The focus indicator should always give a clear center of focus to support visual processing issues. Highlighting something like a chevron that is to the side of the focus area probably isn't enough and may even be out of view if the screen is magnified or the user has limited vision.  
    - At a minimum, WCAG 3 should say using a complete bounding box is a best practice for keyboard focus indicators. Is a complete bounding box essential for users with limited vision or visual processing impairment?

**COGA user needs missing or not adequately addressed - from Making Content Usable for People with Cognitive and Learning Disabilities (MCU):** 

- I need to know how to use all the controls and the effects of each action. ([MCU 3.1.2](https://www.w3.org/TR/coga-usable/#clear-operation-user-story), bullet 2)
- I need to know what happens when I touch things. I know the consequence of each action, such as sending information, changing settings, changing the context or closing the application. (MCU 3.1.2, bullet 6)

With thanks,
The COGA Task Force
